README.md

```diff
@@ -44,6 +44,28 @@ Your Fomu should be running Foboot v1.8.7 or newer. You can see what version you
 
 Aside from that, you need a computer with a USB port that can run the toolchain software. You should need any special drivers, though on Linux you may need sudo access, or special udev rules to grant permission to use the USB device from a non-privileged account.
 
+### Setting udev permissions
+
+```
+sudo groupadd plugdev
+sudo usermod -a -G plugdev $USER
+```
+
+You will need to log out and log in again in order to be a member of the plugdev group.
+
+Create a file named ```/etc/udev/rules.d/99-fomu.rules``` and add the
+following:
+```
+SUBSYSTEM=="usb", ATTRS{idVendor}=="1209", ATTRS{idProduct}=="5bf0", MODE="0664", GROUP="plugdev"
+```
+
+You can reload the udev-rules using the following:
+```
+sudo udevadm control --reload-rules
+sudo udevadm trigger
+```
```

Comment:
I think we want something like `ls /dev/usb/???` here to check the group on the device?